src/sentry/incidents/endpoints/serializers/alert_rule.py

```diff
@@ -272,6 +273,10 @@ def serialize(
         aggregate = translate_aggregate_field(
             obj.snuba_query.aggregate, reverse=True, allow_mri=allow_mri
         )
+        condition_type = None
+        if obj.activation_condition.exists():
+            activation_condition = obj.activation_condition.first()
+            condition_type = activation_condition.condition_type if activation_condition else None
```

Comment:
might be a little cleaner to use `getattr` here:
```suggestion
            condition_type = getattr(activation_condition, 'condition_type', None)
```

If you wanted to get a bit fancier you can merge lines 278 and 279 like this:
```py
condition_type = obj.activation_condition.values_list('condition_type', flat=True).first()
```

This works by using [`values_list`](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#django.db.models.query.QuerySet.values_list) to limit the properties to `condition_type` and return the first one, by default then it'd return `None` from that.

Reply:
Oh nice i like that 🙌